### PR TITLE
 Used SVG fundraising heart. 

### PR DESCRIPTION
**Pull request by codingjoe**
It currently looks very pixelated on retina displays
Screenshot 2019-07-09 at 20 01 35

### CODE REVIEW

**Comment by felixxm** [comment]
After: 
image

File: djangoproject/static/img/fundraising-heart.svg [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<svg viewBox="0, 0, 350, 300" xmlns="http://www.w3.org/2000/svg" width="350" height="300">
+    <rect x="50" width="50" height="50" fill="#ad1d44" stroke="none"/>
+    <rect x="100" width="50" height="50" fill="#d9185c" stroke="none"/>
+    <rect x="200" width="50" height="50" fill="#d9185c" stroke="none"/>
```
Comment (felixxm) [inline djangoproject/static/img/fundraising-heart.svg]: We use `#d9195c`, you can check current colors in [fundraising-heart.js](https://github.com/django/djangoproject.com/blob/master/djangoproject/static/js/mod/fundraising-heart.js).

**Comment by felixxm** [comment]
I pushed colors edits.

**Comment by codingjoe** [comment]
@carltongibson yes, I handcoded it ;)
@felixxm thx :)

**Pull request merged**